CRM/Upgrade/Snapshot.php

```diff
@@ -133,8 +133,6 @@ public static function createTableName(string $owner, string $version, string $n
    *   Ex: '5.50'
    * @param string $name
    * @param \CRM_Utils_SQL_Select $select
-   *
-   * @return iterable
```

Comment:
If this needs to go then the typehint is wrong too

Reply:
No, the type-hint is correct. The `@return` was correct, too. As discussed in the git commit message, the problem is a bug in the phpcs rule. But I didn't want your entire PR blocked on figuring out a bugfix for phpcs, and it's preferrable to have *some* indication of the return type.